MPSGameLift/Scripts/export_gamelift_server_package.py

```diff
@@ -10,10 +10,14 @@
 
 To use this script pass it into o3de.bat's export-project command: 
 <path-to-o3de-engine>\scripts\o3de.bat export-project -es <path-to-multiplayer-sample>\MPSGameLift\Scripts\export_gamelift_server_package.py -ll INFO
+A folder named GameLiftPackageWindows containing the server will be created. 
+Test the server locally and upload the server package to GameLift. 
+    'aws gamelift upload-build --server-sdk-version 5.0.0 --operating-system WINDOWS_2016 --build-root .\GameLiftPackageWindows\ --name MultiplayerSample --build-version v1.0 --region us-west-2'
```

Comment:
I would recommend echoing these notes at the end of the build process as well for new folks to follow next steps easier. either that, or provide a subcommand in this script that also runs this command for those setup.

Reply:
Log added (see end of file)